src/main/nsis/bin/process.bat

```diff
@@ -24,6 +24,25 @@ goto end
 set PROCESS_OPTION=
 if not [%2]==[] set PROCESS_OPTION=process.name=%2
 
-..\Java\bin\java.exe -cp ..\${pom.build.finalName}-uber.jar -Dsalesforce.config.dir=%1 com.salesforce.dataloader.process.ProcessRunner %PROCESS_OPTION%
+
+IF "%JAVA_HOME%" == "" (
+    for /f "tokens=*" %%i in ('${pom.build.finalName}-java-home.exe') do (
+        IF EXIST "%%i" (
+            set JAVA_HOME=%%i
+        ) ELSE (
+            echo %%i
+        )
+    )
+)
+
+IF "%JAVA_HOME%" == "" (
+    echo Please set JAVA_HOME before running the encrypt.bat
```

Comment:
If anything, it seems to add more help. 
I'll approve the merge request and you can make the edits based on Tess's feedback.